### PR TITLE
Step: 1.8.0-SNAPSHOT -> 1.9.0

### PR DESCRIPTION
**Pull request by AlvaroVega**
Mimics PR https://github.com/telefonicaid/perseo-core/pull/180

### CODE REVIEW

File: perseo-main/pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <parent>
```
Comment (fgalan) [inline perseo-main/pom.xml:24]: Looking to reference PR https://github.com/telefonicaid/perseo-core/pull/180, CNR should be flushed in this PR.

File: rpm/SPECS/cep-core.spec [hunk 2]
```diff
@@ -102,6 +102,14 @@ rm -rf $RPM_BUILD_ROOT
 %{_install_dir}/perseo-core.war
 
 %changelog
+* Mon Mar 14 2022 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 1.9.0
+- ADD: unhardwire default internal timer msec resolution (millisecond resolution of the internal timer thread) with a default of 10ms (#194)
+- ADD: INTERNAL_TIMER_MSEC_RESOLUTION env var for internal msec resolution (#194)
+- FIX: Ensure timerules are stored with unique name by using full name which includes service and subservice (#191)
+- FIX: upgrade docker based image from Tomcat8 to Tomcat9
+- FIX: migrate log4j v1 (1.7.25) to v2 (2.17.2) (#184)
+- ADD: allow use WARN as WARNING log level
```
Comment (fgalan) [inline rpm/SPECS/cep-core.spec:111]:
```suggestion
* Tue Mar 15 2022 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 1.9.0
- ADD: unhardwire default internal timer msec resolution (millisecond resolution of the internal timer thread) with a default of 10ms (#194)
- ADD: INTERNAL_TIMER_MSEC_RESOLUTION env var for internal msec resolution (#194)
- ADD: allow use WARN as WARNING log level
- FIX: Ensure timerules are stored with unique name by using full name which includes service and subservice (#191)
- FIX: upgrade docker based image from Tomcat8 to Tomcat9
- FIX: migrate log4j v1 (1.7.25) to v2 (2.17.2) (#184)
```

File: perseo-main/pom.xml (hunk 1 above)
Comment (AlvaroVega) [reply perseo-main/pom.xml:24]: Done in https://github.com/telefonicaid/perseo-core/pull/198/commits/5b7129e164ad0e83d9e1154d61acbb5277d3021e

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**